dev/CommonStyles/FlipView_themeresources.xaml

```diff
@@ -7,14 +7,20 @@
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
+            <x:Double x:Key="FlipViewButtonFontSize">8</x:Double>
+            <x:Double x:Key="FlipViewButtonFontSizePointerOver">9</x:Double>
+            <x:Double x:Key="FlipViewButtonFontSizePressed">7</x:Double>
             <Thickness x:Key="FlipViewButtonBorderThemeThickness">0</Thickness>
```

Comment:
These resources are duplicated three times for all the themes - you could just extract them out of the ThemeDictionaries  to avoid duplication :)